src/components/UnauthorizedError.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+const UnauthorizedError = ({ message }) => {
+  return (
+    <div className="w-screen h-screen flex flex-col items-center justify-center fixed">
+      <p className="text-center text-6xl font-extrabold text-hackathon-blue-100 m-0">
+        401
```

Comment:
401 is when they haven't signed in, 403 is when they dont have admin/required permissions